apps/cloudinary/src/index.js

```diff
@@ -158,13 +163,23 @@ function validateParameters(parameters) {
     return 'Provide your Cloudinary API key.';
   }
 
-  const validFormat = /^[1-9][0-9]*$/.test(parameters.maxFiles);
-  const int = parseInt(parameters.maxFiles, 10);
-  const valid = validFormat && int > 0 && int <= MAX_FILES_UPPER_LIMIT;
-  if (!valid) {
+  const validMaxFilesFormat = /^[1-9][0-9]*$/.test(parameters.maxFiles);
+  const maxFiles = parseInt(parameters.maxFiles, 10);
+  const validMaxFiles = validMaxFilesFormat && maxFiles > 0 && maxFiles <= MAX_FILES_UPPER_LIMIT;
+  if (!validMaxFiles) {
     return `Max files should be a number between 1 and ${MAX_FILES_UPPER_LIMIT}.`;
   }
 
+  let defaultTransformations = [];
+  try {
+    defaultTransformations = JSON.parse(parameters.defaultTransformations);
+  } catch(err) {
+    defaultTransformations = err;
+  }
+  if (!Array.isArray(defaultTransformations)) {
+    return 'Default transformations should be a valid JSON array.';
+  }
```

Comment:
I'd rather return the error string in the catch block to make this behaviour more predictable.
```suggestion
  } catch(err) {
    return 'Default transformations should be a valid JSON array.';
  }
```